YukonVaccinationVerifier/YukonVaccinationVerifier/ViewController.swift

```diff
@@ -320,6 +320,10 @@ class ViewController: UIViewController {
         view.addSubview(logoImageView)
         
         logoImageView.image = UIImage(named: "onCameraLogo")
+        
+        // Add Business Guidance Links
+        
+        BusinessGuidanceViewBuilder.setupView(in: self, links: [BusinessGuidanceViewBuilder.BusinessGuidanceLink(title: "COVID-19 information", url: "https://yukon.ca/en/covid-19-information"), BusinessGuidanceViewBuilder.BusinessGuidanceLink(title: "Data collection notice", url: nil)])
```

Comment:
This function has the tendency to get called more than once.
Current implementation suggests that this 'BusinessGuidanceViewBuilder' will stack up a UIView for every invocation.
I don't agree with your reference of 'view.addSubview(logoImageView)'. That's implemented differently.
Please test this to get a better understanding.

Reply:
I have checked and it's never called again even after presenting the QR check result screen and dismissing back to the scanner screen.
However, I have guarded creating this view twice by adding a tag for it and checking if any view exist with that tag.